### ISSUE TITLE
Syntax error - invalid character

### ISSUE THREAD

**Issue by jimithing78** [astuder/lightroom-map-fix#38]
Hi, I'm trying to follow your guide and I've got my API key created, exported the .LUA files to the same directory as the patchluastr.py but I keep getting the syntax error below

  File "C:\Test\patchluastr.py", line 107
    <title>lightroom-map-fix/patchluastr.py at master · astuder/lightroom-map-fix · GitHub</title>
                                                      ^
SyntaxError: invalid character '·' (U+00B7)

I've opened the file in Python and see the line it's talking about but I know nothing about Python so I'm not sure what the syntax should be.  Any suggestions?  Thanks.

**Comment by vb0** [comment]
Sounds like you downloaded the html view of the .py file and you actually have some html file intended to be shown in browser. Click on "raw" first to get the real file (should have absolutely no html formatting).  

It's something common with github, Linus Tech Tips YouTube channel has some "Linux challenge" and they noted one of the quirks and easy mistakes people inevitably run into first when using github.

**Comment by jimithing78** [comment]
Thank you, you were correct.  I clicked "raw" which showed the content of the file in my browser.  Then I went to my browser window and clicked "Save As" and made sure the extension was set to .py.  I was able to run the commands through the prompt.  Lightroom is going through the "Address Lookup" now on all my pictures.  Hopefully it'll run through the Facial Recognition next, which is what I'm really after.  Thank you again for your reply.
